workflows/data_pipelines/elasticsearch/es_fields.py

```diff
@@ -1,7 +1,6 @@
 def get_elasticsearch_field_name(param_name: str) -> str:
     corresponding_es_field = {
         "liste_idcc": "convention_collective_renseignee",
-        "liste_finess": "est_finess",
```

Comment:
Don't use need `liste_finess_geographique` in this mapping that you use in line 567 in file `workflows/data_pipelines/elasticsearch/data_enrichment.py`?

Reply:
or delete it beause of `workflows/data_pipelines/elasticsearch/process_unites_legales.py` line 244?